src/ptvsd/adapter/messages.py

```diff
@@ -155,8 +155,12 @@ def attach_request(self, request):
         self.terminate_on_disconnect = False
         self._debug_config(request)
 
-        # TODO: get address and port
-        channels.connect_to_server()
+        host = request.arguments.get("host", constants.DEFAULT_DEBUG_SERVER_HOST)
+        port = request.arguments.get("port", constants.DEFAULT_DEBUG_SERVER_PORT)
```

Comment:
To mirror the pattern in `ptvsd.common` and `ptvsd.server`, let's add `ptvsd.adapter.options` to this package along the same lines. Then all the defaults can go there - and the code here will overwrite them if they were supplied in the message, but then just use `options.port` etc to connect.